src/Core/Services/StateService.cs

```diff
@@ -650,23 +645,18 @@ public async Task SetLastBuildAsync(string value)
             await SetValueAsync(key, value, options);
         }
 
-        public async Task<bool?> GetDisableFaviconAsync(string userId = null)
+        public async Task<bool?> GetDisableFaviconAsync()
         {
-            var reconciledOptions = ReconcileOptions(new StorageOptions { UserId = userId },
-                await GetDefaultStorageOptionsAsync());
-            var key = Constants.DisableFaviconKey(reconciledOptions.UserId);
-            return await GetValueAsync<bool?>(key, reconciledOptions);
+            var options = await GetDefaultStorageOptionsAsync();
+            var key = Constants.DisableFaviconKey;
+            return await GetValueAsync<bool?>(key, options);
```

Comment:
I would avoid assigning the `key` line and just use the `Constants.DisableFaviconKey` inline in the func. Furthermore, it has `Key` as suffix so it's obvious that it's a key in the parameter so we don't lose that either when reading it.

Reply:
If it's ok I'd like to add this (and the others mentioned below) to the followup PR mentioned in the description.  It's a pattern used throughout all of `StateService` so changing it will be noisy.